src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -22,10 +23,20 @@
 def get_file_paths(directory):
     logger.debug(locals())
     file_paths = []
-    for root, _, files in os.walk(directory):
-        for file in files:
-            if file.split("/")[-1] == "qna.yaml":
-                file_paths.append(os.path.join(root, file))
+    root_paths = [
+        entry
+        for entry in Path(directory).iterdir()
+        if entry.is_dir()
+        if not entry.name.startswith(".")
+        if entry.name != "knowledge"
+        if entry.name != "docs"
+        if entry.name != "scripts"
+    ]
+    for basedir in root_paths:
+        for root, _, files in os.walk(basedir):
+            file_paths.extend(
+                [os.path.join(root, file) for file in files if file == "qna.yaml"]
```

Comment:
Observation. I suspect this whole `walk`ing / `extend`ing business could be replaced with a simple `glog.glob` but I don't think we should care now, if at all.